File: acm_statistic_backend/api/api/DTO/passager/Passager_detail.cs
namespace api.DTO.passager
{
    public class Passager_detail
    {
        public int Mois { get; set; }
        public string Mois_lettre { get; set; }

        public int Annee { get; set; }

        public String? Type_trafic { get; set; }
        public decimal? Montant { get; set; }
        public string? Compagnie { get; set; }
        public string? Aeroport { get; set; }
        public int Passagers { get; set; }
        public int Vols { get; set; }
    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/passager/PassagerDetail.ts
export class PassagerDetail{
    mois: number;
    mois_lettre: string | null;
    annee :number; 
    type_trafic: string | null;
    montant: number;
    compagnie: string | null;
    aeroport: string | null;
    passagers: number;
    vols: number;

    constructor(init?: Partial<PassagerDetail>) {
    Object.assign(this, init);
    }

}